### PR TITLE
update alphanet relay chain endpoint

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

This PR updates the alphanet endpoint.

These ones are being decommissioned: frag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network , frag-moonbase-relay-rpc.g.moonbase.moonbeam.network . The replacement endpoint is this one: fro-moon-rpc-1-moonbase-relay-rpc-1.moonbase.ol-infra.network

Goes with: https://github.com/moonbeam-foundation/moonbeam-docs/pull/905

### CODE REVIEW

File: .snippets/text/builders/get-started/endpoints/moonbase.md [hunk 1]
```diff
@@ -20,4 +20,4 @@
 
 |  供应商   |                                    RPC URL                                    |
 |:---------:|:-----------------------------------------------------------------------------:|
-| PureStake | <pre>```wss://frag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network```</pre> |
\ No newline at end of file
+| PureStake | <pre>```wss://fro-moon-rpc-1-moonbase-relay-rpc-1.moonbase.ol-infra.network```</pre> |
```
Comment (albertov19) [inline .snippets/text/builders/get-started/endpoints/moonbase.md:23]: Should we change PureStake?

**Review by albertov19 — DISMISSED** [review]
LGTM!

File: learn/platform/networks/moonriver.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ description: Moonriver（Moonbeam在Kusama上部署的平行链）的当前配
 目前，Moonriver具有以下配置：
 
 - 以平行链的形式运行并连接至Kusama中继链
-- 拥有一个由PureStake代表Moonbeam基金会运营并含有{{ networks.moonriver.staking.max_candidates }}位收集人的有效集。
```
Comment (albertov19) [inline learn/platform/networks/moonriver.md:19]: Here you removed PureStake but did not add OpsLayer? I mean I don't speak Chinese but unsure haha

File: learn/platform/networks/overview.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ description: 基于Moonbeam的所有MainNet和TestNet部署概述；Moonbeam是
 
  - Moonbeam: 部署于Polkadot上（2021 十二月）
  - Moonriver: 部署于Kusama上（2021 六月）
- - Moonbase Alpha: PureStake主导平行链测试网(2020 九月)
+ - Moonbase Alpha: Moonbeam 和 Moonriver 的平行链测试网(2020 九月)
```
Comment (albertov19) [inline learn/platform/networks/overview.md:14]: Same here

**Review by albertov19 — CHANGES_REQUESTED** [review]

File: learn/platform/networks/moonriver.md (hunk 2 above)
Comment (eshaben) [reply learn/platform/networks/moonriver.md:19]: It was outdated. You can reference the Moonbeam page and see what I mean or the Moonriver page on the English site

File: learn/platform/networks/overview.md (hunk 3 above)
Comment (eshaben) [reply learn/platform/networks/overview.md:14]: It was outdated. You can reference the Overview page on the English site

**Review by albertov19 — APPROVED** [review]

**Pull request merged**